tests/signUp/signUp.spec.js

```diff
@@ -0,0 +1,286 @@
+import {test, expect} from "@playwright/test";
+
+test.describe("Positive flow", ()=>{
+    test.beforeEach(async ({page})=>{
+        await page.goto("")
+        const signUpButton = page.locator('button', {hasText: 'Sign Up'})
+        await signUpButton.click()
+    })
+    test ('Sign Up with VALID data', async ({page})=>{
+        const popup = page.locator('app-signup-modal')
+
+        const inputName = popup.locator('#signupName')
+        await expect(inputName).toBeVisible()
+        await expect(inputName).toBeEnabled()
+        await expect(inputName).toBeEditable()
+
+        const inputLastName = popup.locator('#signupLastName')
+        await expect(inputLastName).toBeVisible()
+        await expect(inputLastName).toBeEnabled()
+        await expect(inputLastName).toBeEditable()
+
+        const inputEmail = popup.locator('#signupEmail')
+        await expect(inputEmail).toBeVisible()
+        await expect(inputEmail).toBeEnabled()
+        await expect(inputEmail).toBeEditable()
+
+        const inputPassword = popup.locator('#signupPassword')
+        await expect(inputPassword).toBeVisible()
+        await expect(inputPassword).toBeEnabled()
+        await expect(inputPassword).toBeEditable()
+
+        const inputRepeatPassword = popup.locator('#signupRepeatPassword')
+        await expect(inputRepeatPassword).toBeVisible()
+        await expect(inputRepeatPassword).toBeEnabled()
+        await expect(inputRepeatPassword).toBeEditable()
+
+        const submitBtn = popup.locator('button', {hasText: 'Register'})
+        await expect(submitBtn).toBeVisible()
+        await expect(submitBtn).toBeDisabled()
+
+        await inputName.fill('Ivanna')
+        await inputLastName.fill('Dub')
+        await inputEmail.fill('aqa-testing.idub@gmail.com')
+        await inputPassword.fill('12345qwerrtyY!')
+        await inputRepeatPassword.fill('12345qwerrtyY!')
+        await submitBtn.click()
```

Comment:
you need to ensure, your test was successful, check url or assert some element is visible